cli/src/main/java/org/apache/falcon/cli/FalconEntityCLI.java

```diff
@@ -329,13 +362,13 @@ public void entityCommand(CommandLine commandLine, FalconClient client) throws F
         OUT.get().println(result);
     }
 
-    private void validateColo(Set<String> optionsList) throws FalconCLIException {
+    private void validateColo(Set<String> optionsList) {
```

Comment:
This method throws FalconCLIException. Why is it removed from method signature? Same applies for most methods below.